src/View/Legale/legale.html.twig

```diff
@@ -1,5 +1,6 @@
 {% extends 'layout.html.twig' %}
-
+	{% block title %}Mention légale
```

Comment:
Mention"s" légales 
and as said for other views, keep main title with parent injection